0.data_analysis/plate_5_characteristics/utils/CorrelateAggWells.py

```diff
@@ -0,0 +1,129 @@
+from collections import defaultdict
+from itertools import combinations, product
+
+import pandas as pd
+
+
+class CorrelateAggWells:
```

Comment:
if there are no class attributes being defined in the methods, consider taking a functional approach and removing the class. I might be missing something but is there a reason for an OOP approach vs using helper functions???

Reply:
Good question, I was originally going to use class attributes, but decided my code would be more modular if I didn't. Class members might improve the computational efficiency, but the user would have to instantiate multiple objects to compute correlations for different datasets. I may change this in the future, in general though, I like the encapsulations and extensibility of classes better than functions defined in a separate python file.